core/lib/default_da_clients/src/no_da/client.rs

```diff
@@ -0,0 +1,36 @@
+use std::fmt::Debug;
```

Comment:
Nit: AFAICT, this import can be removed; it's not used as a trait, and `Debug` as a derive macro is imported automatically (i.e., it's a part of the `std` prelude).